cedar-policy-core/cedar-policy-macros/tests/fail.rs

```diff
@@ -0,0 +1,8 @@
+use cedar_policy_core::ast::EntityUID;
+use cedar_policy_macros::euid;
+
+fn main() {
+    let _: EntityUID = euid!(f, "bar");
+    let _: EntityUID = euid!(Foo::Bar, "baz");
+    let _: EntityUID = euid!(Foo, "ha");
+}
```

Comment:
Why is this file called `fail.rs`? what about these examples fails?

Reply:
This is actually related to your second question above. This file should contain macro usages that do not compile. However, it seems cargo will try to build each file in this package, which contains this file that is supposed to fail to compile.